include/dem/dem_properties.h

```diff
@@ -37,28 +37,30 @@ namespace DEM
 
   enum PropertiesIndex : int
   {
-    id           = 0,
-    type         = 1,
-    dp           = 2,
-    rho          = 3,
-    v_x          = 4,
-    v_y          = 5,
-    v_z          = 6,
-    acc_x        = 7,
-    acc_y        = 8,
-    acc_z        = 9,
-    force_x      = 10,
-    force_y      = 11,
-    force_z      = 12,
-    omega_x      = 13,
-    omega_y      = 14,
-    omega_z      = 15,
-    mass         = 16,
-    mom_inertia  = 17,
-    M_x          = 18,
-    M_y          = 19,
-    M_z          = 20,
-    n_properties = 21
+    type           = 0,
+    dp             = 1,
+    rho            = 2,
+    v_x            = 3,
+    v_y            = 4,
+    v_z            = 5,
+    acc_x          = 6,
+    acc_y          = 7,
+    acc_z          = 8,
+    force_x        = 9,
+    force_y        = 10,
+    force_z        = 11,
+    omega_x        = 12,
+    omega_y        = 13,
+    omega_z        = 14,
+    mass           = 15,
+    mom_inertia    = 16,
+    M_x            = 17,
+    M_y            = 18,
+    M_z            = 19,
+    displacement_x = 20,
+    displacement_y = 21,
+    displacement_z = 22,
```

Comment:
Why store each component of the displacement? storing the total displacement would be sufficient (and cheaper)